packages/rancher-windows-exporter/charts/templates/daemonset.yaml

```diff
@@ -45,7 +45,7 @@ spec:
         - name: wins-pipe
           mountPath: \\.\pipe\rancher_wins
         - name: binary-host-path
-          mountPath: c:/host/etc/windows-exporter
+          mountPath: {{ regexReplaceAll ".:/" .Values.global.cattle.rkeWindowsPathPrefix  "${1}C:/host/"}}etc/windows-exporter
```

Comment:
No need to do the `regexReplaceAll` here; `C:/host/etc/windows-exporter` can be hard-coded.

This is the same logic for why we didn't need to modify the [mountPath in rancher-wins-upgrader](https://github.com/rancher/wins/blob/4bf76f4ecbc0ea7b835a4080d36705737a41559a/charts/rancher-wins-upgrader/templates/daemonset.yaml#L40); this path is just what the **container** sees (and what the scripts reference).

All that needs to be changed is the directory that this containerPath points to; e.g. the hostPath modified below.